packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -71,7 +71,7 @@ export class MeetWidget extends Component {
               isActive={isActive}
               localVideoPosition={localVideoPosition}
               onHangupClick={handleHangup}
-              {...call}
+              {...call.toJS()}
```

Comment:
I've been trying to keep all of the immutable conversions out of containers/renders. Could this go in the selector instead?

Reply:
Unfortunately not without a big refactor. There are HOCs that need the call object to still be a immutable record and not a plain js object.

Reply:
Ah, ok, and this worked previously because the call object was no longer an immutable object (which is the bug you fixed).